scripts/webpack.js

```diff
@@ -20,13 +20,11 @@ const rules = () => ({
       },
     },
   }),
-  css: () => [
-    {
-      test: /\.css$/,
-      include: ['ol', 'redux-notifications', 'react-datetime', 'codemirror'].map(moduleNameToPath),
-      use: ['to-string-loader', 'css-loader'],
-    },
-  ],
+  css: () => ({
```

Comment:
Is this required? Looks like `'to-string-loader'` was removed

Reply:
No it's not - reverted.